shopinvader/tests/test_customer.py

```diff
@@ -31,6 +40,11 @@ def setUp(self, *args, **kwargs):
             self.service = work.component(usage="customer")
             self.address_service = work.component(usage="addresses")
 
+        with self.work_on_services(
```

Comment:
not that you could skip this by using `self.service._load_partner_work_context(self.partner_binding)` to switch partner.
Not a blocker of course.